peac/cpp/plane_fitter_pcd.cpp

```diff
@@ -282,72 +281,37 @@ int process() {
 	bool is_debug = global::iniGet<bool>("debug", 0);
 	int loop_cnt = global::iniGet<int>("loop", 0); //0: no loop; -1: infinite loop; >0: n loops
 
-	std::vector<std::string> fnamelist;
-	std::string filelist = global::iniGet<std::string>("list","list.txt");
-
-	if (filelist.find(".pcd") != std::string::npos) {
-		fnamelist.push_back(filelist); //support list=test.pcd to only input a single file
-		loop_cnt = global::iniGet<int>("loop", -1); //by default turn on infinite loop (-1) mode
-	} else {
-		std::string inputDir = global::getFileDir(filelist);
-
-		std::ifstream is(filelist.c_str());
-		if (!is.is_open()) {
-			std::cout << "could not open list=" << filelist << std::endl;
-			return -1;
-		}
-
-		while (is) {
-			std::string fname;
-			std::getline(is, fname);
-			if (fname.empty()) continue;
-			fname = inputDir + global::filesep + fname;
-			fnamelist.push_back(fname);
-		}
-	}
 
 	using global::showWindow;
-	showWindow = global::iniGet("showWindow", true);
+	showWindow = global::iniGet("showWindow", false);
 	if (showWindow)
 		cv::namedWindow("seg");
-
-	int idx = 0;
-	while(true)
-	{
-		if (idx >= fnamelist.size()) {
-			if (loop_cnt != 0) {
-				idx = 0;
-				if (loop_cnt > 0) --loop_cnt;
-			}
-			else break;
-		}
-
-		const std::string& fname = fnamelist[idx];
-		pcl::PointCloud<pcl::PointXYZ> cloud;
-		if(pcl::io::loadPCDFile(fname, cloud) <0) {
-			std::cout<<"fail to load: "<<fname<<std::endl;
-		} else {
-			pcl::transformPointCloud<pcl::PointXYZ>(
-				cloud, cloud,
-				Eigen::Affine3f(Eigen::UniformScaling<float>(
-				(float)unitScaleFactor)));
-			std::cout<<fname<<std::endl;
-
-			std::string outputFilePrefix = outputDir+global::filesep+global::getNameNoExtension(fname);
-			processOneFrame(cloud, outputFilePrefix);
-		}
-
-		if(!is_debug) ++idx;
-		else break; //debug mode always process the first file only and ignore loop
+	
+	std::string input="input/";
+	input+=pcd_filename;
```

Comment:
please, follow c++ codestyle (for example spaces between variables and operators and so on)